apps/website-e2e/src/website.spec.ts

```
@@ -0,0 +1,60 @@
+import type { Browser } from 'playwright'
+import { chromium, expect, test } from '@playwright/test'
+import getPort from 'get-port'
+import { playAudit } from 'playwright-lighthouse'
+
+export const lighthouseTest = test.extend<{ Page }, { port: number, browser: Browser }>({
+  port: [
+    // eslint-disable-next-line no-empty-pattern
+    async ({}, use) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)